src/Chapter1.hs

```diff
@@ -520,7 +523,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if x*x > y*y then y else x
```

Comment:
That is a clever trick, but I guess `abs` would work better here, especially on huge numbers 🙂 This solution potentially could lead to the integer overflow 💥